docs/reference/logging/index.md

```diff
@@ -25,6 +25,16 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
+
+   --log-opt max-size=(unlimited)
+   --log-opt max-file=1
```

Comment:
Thinking of the `max-size` option; if the default is "unlimited", then specifying `max-files` **without** specifying `max-size` is a no-op? I.e., the max-size is never reached, so there will never be a roll-over?